internal/pkg/sessions/mock_store.go

```diff
@@ -41,10 +42,15 @@ func (ms *MockSessionStore) ClearSession(http.ResponseWriter, *http.Request) {
 
 // LoadSession returns the session and a error
 func (ms *MockSessionStore) LoadSession(*http.Request) (*SessionState, error) {
+	if ms.Session == nil {
+		return nil, http.ErrNoCookie
+	}
```

Comment:
pretty trivial but might be cleaner and more concise to to set ms.LoadError to http.ErrNoCookie here?